source/tutorial/kerberos.txt

```diff
@@ -93,12 +107,24 @@ Kerberos authentication on either Windows or macOS.
            :xml:`<mono><link target="https://docs.mongodb.com/manual/reference/configuration-options/#setparameter-option">setParameter</link></mono>`,
            and :xml:`<mono><link target="https://docs.mongodb.com/manual/reference/parameters/#param.authenticationMechanisms">authenticationMechanisms</link></mono>`.
 
-         - Run ``mongod`` as the MongoDB Windows user you 
+         - Run ``mongod`` as the MongoDB Windows user you
            created in Active Directory.
-         - :manual:`Create a user </reference/method/db.createUser>` on your MongoDB database with the 
-           :ref:`appropriate roles <cached-sampling-user-permissions>` 
+         - :manual:`Create a user </reference/method/db.createUser>` on your MongoDB database with the
+           :ref:`appropriate roles <cached-sampling-user-permissions>`
            to sample data.
 
+           .. admonition:: Linux Schema User Authenticating to ADC
+              :class: note
+
+              If you are authenticating a user from a Linux host, the 
+              user must authenticate using Kerberos. On the
+              ``mongod``, this requires:
+
+              - Setting :setting:`mongodb.net.auth.username` to the
+                schema user's |upn|.
+              - Setting :setting:`mongodb.net.auth.password` to no
+                value.
+
```

Comment:
This should most likely be in the BIC configuration section

Reply:
Agreed. Fixed.